src/Track.h

```diff
@@ -367,17 +370,27 @@ class AUDACITY_DLL_API Track /* not final */
 
    std::shared_ptr<TrackList> GetOwner() const { return mList.lock(); }
 
+   LinkType GetLinkType() const noexcept;
+   //! Returns true if next track is linked to this one
+   bool HasLinkedTrack() const noexcept;
```

Comment:
HasLinkedTrack() replaces GetLinked() when used as just a boolean.  I explained that I made sure GetLinked() was a private function when I rewrote all the old code.  HasLinkedTrack() should also be private.  I checked that the code still compiles if it is.